grails-app/services/org/pih/warehouse/core/DocumentTemplateService.groovy

```diff
@@ -130,11 +130,28 @@ class DocumentTemplateService {
                     expectedShippingDate: ""
             ]
         }
+        def groupedOrderItems = activeOrderItems?.groupBy { [product: it.product, unitOfMeasure: it.unitOfMeasure] }?.collect { k, v ->
+            def productSuppliers = v?.productSupplier?.findAll { it }
+            return [
+                code: k.product.productCode,
+                type: "Item",
+                status: v?.orderItemStatusCode?.first() ?: "",
+                description: k.product.name ?: "",
+                supplierCode: productSuppliers?.size() > 0 ? productSuppliers.first()?.supplierCode: "",
+                manufacturer: productSuppliers?.size() > 0 ? productSuppliers.first()?.manufacturer?.name: "",
+                manufacturerCode: productSuppliers?.size() > 0 ? productSuppliers.first()?.manufacturerCode: "",
+                unitOfMeasure: k.unitOfMeasure,
+                unitPrice: v?.unitPrice?.first(),
+                quantity: v?.sum { it.quantity },
+                totalPrice: v?.sum { it.totalPrice() }
+            ]
+        }
 
         def order = orderInstance.toJson()
         order.dateOrdered = orderInstance.dateOrdered
         order.dateApproved = orderInstance.dateApproved
         order.dateCompleted = orderInstance.dateCompleted
+        order.currentDate = new Date()
```

Comment:
:confused: 

This should be added to the context on its own. Call it today or now.